server/apps/distributed-app/crowdsec/scenarios/james-bf-auth.yaml

```diff
@@ -0,0 +1,13 @@
+type: leaky
+name: crowdsecurity/james
+description: "Detect login james bruteforce"
+filter: "evt.Meta.log_type == 'james_failed-auth'"
+leakspeed: "10s"
+capacity: 5
```

Comment:
Is this the configuration for the maximum failure authentication?

e.g. after 5 failure authentication in 10s -> we trigger this scenario.

Reply:
No, there is a queue containing these failure, and its capacity is 5 times. 
- leakspeed means that every 10s, a failure will be released from this queue, like water leaks out the bucket

Reply:
Is it not that within 10 seconds, the maximum failure is 5 (maximum 5 items in the queue)? And for the 6th failure, it trigger the scenario?

Reply:
Obviously, within 10s, if there are 6 failures, it will trigger the scenario. But it is only one case. More generally, whenever the queue gets more than 5 failures, it will trigger the scenario

Reply:
Leaking bucket principle, as I remember, so yes